README.md

```diff
@@ -97,7 +97,7 @@ The following minimal configuration configures the connector with default values
 `"json-values-topic"` with record keys as `String` and `JSON` values (without schema), and sending events to the custom
 EventBridge event bus `"kafkabus"` in region `"us-east-1"`.
 
-```json5
```

Comment:
Why the change :) ?

Reply:
Github has better JSON syntax highlighting when specifying yaml. Please don't ask me why :)

Reply:
<img width="355" alt="image" src="https://github.com/awslabs/eventbridge-kafka-connector/assets/15986659/8e131c4f-cdc7-4018-9735-ea28ac99165f">

Above uses `json5` below uses `yaml`

Reply:
IMHO what you're wondering is why there's no syntax highlighting with `json5` - which is weird, but allows comments in JSON objects. Feel free to change the examples without comments to `json` only for formatting.

Reply:
Done